tools/uefi_builder/common/src/fs.rs

```diff
@@ -159,7 +159,7 @@ pub fn create_fat_filesystem(
         .truncate(true)
         .open(out_fat_path)
         .unwrap();
-    let fat_size_padded_and_rounded = ((needed_size + 1024 * 64 - 1) / MB + 1) * MB;
+    let fat_size_padded_and_rounded = (needed_size / MB + 10) * MB;
     fat_file.set_len(fat_size_padded_and_rounded).unwrap();
```

Comment:
Hey @tsoutsman, this is a crash workaround; I think you wrote the original size computation, could you tell me the reasoning behind it?

Reply:
Modified in #826, tho the fix is still pretty hacky.